Cargo.toml

```diff
@@ -30,21 +38,34 @@ chrono = { version = "0.4.30", default-features = false, optional = true }
 time = { version = "0.3", default-features = false, optional = true }
 futures = { version = "0.3", default-features = false, features = ["std"] }
 log = { version = "0.4", default-features = false }
-tracing = { version = "0.1", default-features = false, features = ["attributes", "log"] }
+tracing = { version = "0.1", default-features = false, features = [
+    "attributes",
+    "log",
+] }
 rust_decimal = { version = "1", default-features = false, optional = true }
 bigdecimal = { version = "0.3", default-features = false, optional = true }
-sea-orm-macros = { version = "0.12.3", path = "sea-orm-macros", default-features = false, features = ["strum"] }
-sea-query = { version = "0.30.2", default-features = false, features = ["thread-safe", "hashable-value", "backend-mysql", "backend-postgres", "backend-sqlite"] }
+sea-orm-macros = { version = "0.12.3", path = "sea-orm-macros", default-features = false, features = [
+    "strum",
+] }
+sea-query = { version = "0.30.2", default-features = false, features = [
+    "thread-safe",
+    "hashable-value",
+    "backend-mysql",
+    "backend-postgres",
+    "backend-sqlite",
+] }
 sea-query-binder = { version = "0.5.0", default-features = false, optional = true }
 strum = { version = "0.25", default-features = false }
 serde = { version = "1.0", default-features = false }
 serde_json = { version = "1.0", default-features = false, optional = true }
-sqlx = { version = "0.7", default-features = false, optional = true }
 uuid = { version = "1", default-features = false, optional = true }
 ouroboros = { version = "0.17", default-features = false }
 url = { version = "2.2", default-features = false }
 thiserror = { version = "1", default-features = false }
 
+[target.'cfg(not(wasm))'.dependencies]
```

Comment:
This position temporarily disables sqlx because rustls, the upstream dependency of sqlx, cannot compile to any wasm target for the time being. The mainly cause is that rustls-dependent rings need to cross-compile code from C language.

Recently, ring may support compilation of the wasm platform by compiling old cipher suites with BoringC instead, but this will take a long time.

Reply:
Related to https://github.com/briansmith/ring/pull/1745